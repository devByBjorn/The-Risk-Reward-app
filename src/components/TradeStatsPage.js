import React from 'react'
import StatsListTrades from './StatsListTrades'
import StatsListR from './StatsListR'
import StatsTable from './StatsTable'
import PageMainContainter from '../components_style/PageMainContainer'
import FlexContainer from '../components_style/FlexContainer'
import Paragrapgh from '../components_style/Paragraph'
import { HorizontalLine } from '../components_style/LineBreak'

const TradeStatsPage = () => (
  <PageMainContainter mobileMargin="10% 1.5rem">
    <FlexContainer mobileDirection="column" justifyContent="space-around">
      <StatsListTrades />
      <StatsListR />
    </FlexContainer>
    <FlexContainer
      justifyContent="space-around" mobileDirection="column">
      <StatsTable keyOfUse='market' />
      <StatsTable keyOfUse='setup' />
    </FlexContainer>
  </PageMainContainter>
)

export default TradeStatsPage