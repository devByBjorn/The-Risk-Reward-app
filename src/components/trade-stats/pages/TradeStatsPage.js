import React from 'react'
import StatsOutcome from '../stats/StatsOutcome'
import StatsR from '../stats/StatsR'
import CountTotalTrades from '../stats/CountTotalTrades'
import StatsSetups from '../stats/StatsSetups'

const TradeStatsPage = () => (
  <React.Fragment>
    <CountTotalTrades />
    <StatsOutcome />
    <StatsR />
    <StatsSetups />
  </React.Fragment>
)

export default TradeStatsPage