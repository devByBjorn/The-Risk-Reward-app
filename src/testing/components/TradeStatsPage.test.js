import React from 'react'
import { shallow } from 'enzyme'
import TradeStatsPage from '../../components/TradeStatsPage'
import { trades } from '../fixtures/trades'

test('should render TradeStatsPage', () => {
  const wrapper = shallow(<TradeStatsPage />)
  expect(wrapper).toMatchSnapshot()
})