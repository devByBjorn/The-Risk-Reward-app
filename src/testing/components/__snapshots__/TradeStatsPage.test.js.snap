// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TradeStatsPage 1`] = `
<PageMainContainer
  mobileMargin="10% 1.5rem"
>
  <FlexContainer
    justifyContent="space-around"
    mobileDirection="column"
  >
    <Connect(StatsListTrades) />
    <Connect(StatsListR) />
  </FlexContainer>
  <FlexContainer
    justifyContent="space-around"
    mobileDirection="column"
  >
    <Connect(StatsTable)
      keyOfUse="market"
    />
    <Connect(StatsTable)
      keyOfUse="setup"
    />
  </FlexContainer>
</PageMainContainer>
`;
